core/infrastructure/kafka/v1beta2/register.go

```diff
@@ -15,6 +15,7 @@
 // NOTE: Boilerplate only.  Ignore this file.
 
 // Package v1beta2 contains API Schema definitions for the kafka v1beta2 API group
+// +kubebuilder:skip
 // +k8s:deepcopy-gen=package,register
 // +groupName=kafka.strimzi.io
 package v1beta2
```

Comment:
maybe evaluate if there is a v1 api available?

Reply:
No, latest Strimzi operator supports API v1beta2